source/_integrations/peco.markdown

```diff
@@ -0,0 +1,22 @@
+---
+title: "PECO Outage Count"
+description: "Get how many outages are in your county"
+ha_release: "2022.2.0"
+ha_category: Sensor
+ha_iot_class: "Cloud Polling"
+ha_quality_scale: platinum
+ha_config_flow: true
+ha_codeowners:
+  - '@IceBotYT'
+ha_domain: peco
+---
+
```

Comment:
- Can we add something like links to the sources too?
- Maybe extend the story a little. For example, what this integration can do/is useful for.
- A comment for the countries this applies to might be helpful. I'm pretty sure the counties in my country, aren't covered 😉